shared/src/main/scala/org/atnos/eff/Eff.scala

```diff
@@ -138,13 +144,14 @@ object Eff extends EffCreation with
 trait EffImplicits {
 
   /**
-   * Monad implementation for the Eff[R, ?] type
-   */
-  implicit final def EffMonad[R]: Monad[Eff[R, ?]] = new Monad[Eff[R, ?]] {
-    def pure[A](a: A): Eff[R, A] =
+    * Monad implementation for the Eff[R, ?] type
+    */
+  final val effMonadUnsafe: Monad[Eff[AnyRef, ?]] = new Monad[Eff[AnyRef, ?]] {
```

Comment:
Yes, that would be better, I didn't realize that was the intent with `unsafe` in the name.